docs/how_to_create_a_plugin.md

```diff
@@ -1,11 +1,12 @@
 # How to create a Decidim engine
 
 1. Run the following command:
+
 ```bash
 - rails plugin new decidim-<engine_name>
 ```
 
-2. Change the `decidim-<engine_name>.gemspec` file:
+1. Change the `decidim-<engine_name>.gemspec` file:
```

Comment:
When adding a bunch of markdown code between list elements, markdown "breaks" the list, so every time you add `1.` it is considered a new list, so this is not working :(